tests/test.py

```diff
@@ -798,6 +801,46 @@ def run_terraform(*, image: str, final: bool = False):
 
     num_tests_ran += num_successful_tests
 
+    # Run secondary interactive terraform tests
+    test_interactive_container = CLIENT.containers.run(
+        image=image,
+        detach=True,
+        auto_remove=False,
+        tty=True,
+        volumes=secure_volumes_with_log_config,
+        environment=environment,
+    )
+
+    # Running an interactive terraform command
+    test_interactive_container.exec_run(
+        cmd='/bin/bash -ic "terraform validate"', tty=True
+    )
+
+    # An interactive terraform command should still cause the creation of the following files, and should have the same number of logs lines in the
+    # fluent bit log regardless of which image is being tested
+    files = [str(kics_output_file)]
+    files.append(str(checkov_output_file))
+    LOG.debug("Testing that interactive terraform commands still create json reports")
+    number_of_security_tools = len(CONFIG["commands"]["terraform"]["security"])
+    expected_number_of_logs = number_of_security_tools
+
+    # Check the container
+    if (
+        num_successful_tests := check_container(
+            container=test_interactive_container,
+            files=files,
+            files_expected_to_exist=True,
+            log_path="/tmp/fluent_bit.log",
```

Comment:
This is correct because when running the tests we volume mount the `tests/fluent-bit.outputs.conf` into the container, and that config says to write logs to `/tmp/fluent_bit.log`.  This is a way to test that fluent-bit is functional, because it pulls logs from `/var/log/easy_infra.log` and writes them to `/tmp/fluent_bit.log`.